spec/support/pages/forgot_password.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class ForgotPassword < SitePrism::Page
+  set_url '/users/password/new'
+
+  element :forgot_title, 'h1'
```

Comment:
If there is a few h1 tags?
```suggestion
  element :forgot_title, 'h1', text: I18n.t(:forgot_password_title)
```